editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -32,8 +33,20 @@ interface ComputedLook {
 
 export const BasePaddingUnit = 20
 
-export function getElementPadding(elementPath: ElementPath): number {
-  return EP.navigatorDepth(elementPath) * BasePaddingUnit
+export function getElementPadding(
+  elementPath: ElementPath,
+  visibleNavigatorTargets: Array<ElementPath>, // TODO: we only need this to filter out fragments, can be delete after 'Fragment support' FS is deleted
```

Comment:
Can we make sure this is being tracked in the paper doc please

Reply:
Ok, done!
<img width="742" alt="image" src="https://user-images.githubusercontent.com/127662/218443480-2a525bbe-0dc4-4624-9fe4-925c97c0a974.png">